app/packages/state/src/recoil/distributions.ts

```diff
@@ -35,11 +35,26 @@ import {
   withPath,
 } from "@fiftyone/utilities";
 import { State } from "./types";
+import { extendedSelection } from "./atoms";
+import { filters } from "./filters";
+import { groupSlice, groupStatistics } from "./groups";
 
 export type AggregationResponseFrom<
   TAggregate extends { response: { aggregate: readonly unknown[] } }
 > = TAggregate["response"]["aggregate"][0];
 
+const extendedViewForm = selector({
```

Comment:
Does this have any relation to the StateForm.extended?

Reply:
Not too sure if it is related to the `extended` property, but there is much overlap between StateForm and ExtendedViewForm.

Reply:
They are definitely similar. They may be the exact same, but an audit would have to be done to be sure.